get_question_sharing_info.sql

```diff
@@ -0,0 +1,34 @@
+-- docker exec -it mypl psql postgres -f PrairieLearn/get_question_sharing_info.sql > question_sharing_info.csv
+COPY (
+  SELECT
+    *
+  FROM
+    (
+      SELECT
+        q.qid,
+        q.shared_publicly,
+        (
+          SELECT
+            jsonb_agg(ss.name)
+          FROM
+            sharing_set_questions AS ssq
+            JOIN sharing_sets AS ss on (ss.id = ssq.sharing_set_id)
+          WHERE
+            ssq.question_id = q.id
+        ) AS sharing_sets
+      FROM
+        questions AS q
+      WHERE
+        q.course_id = 2
```

Comment:
There are currently 48 courses with question sharing enabled. Manually running this and the other query 100 times and subsequently SCPing those 100 files will take a lot of time. Consider writing this as a query that uses `feature_grants` to identify the appropriate courses and includes the course ID/name/owners in the CSV. A small amount of Python should allow you to post-process that into separate CSVs per course.

Reply:
48???? I thought it was only like <10 a few months ago. 

Anyway, why do we need to check feature grants? We can just get rid of the `where course_id = ` clause. Nothing will be in the sharing related tables for courses that don't have it enabled.

Reply:
> I thought it was only like <10 a few months ago.

It was, the feature is in high demand 😉 

I think you're correct that we don't need to check the feature flag, we can just look at any shared questions that exist.

Reply:
fixed